src/layers/core/grid-layer/grid-layer.js

```diff
@@ -89,10 +89,14 @@ export default class GridLayer extends Layer {
 
     const {attributeManager} = this.state;
     attributeManager.addInstanced({
-      instancePositions: {size: 4, update: this.calculateInstancePositions},
+      instancePositions: {
+        size: 4,
+        update: this.calculateInstancePositions
```

Comment:
Same as ArcLayer here - GridLayer use multiple accessors to generate instancePositions.